src/fluent-plugin-mdsd/test/plugin/test_out_mdsd.rb

```diff
@@ -48,6 +49,7 @@ def test_configure()
         assert_equal("/tmp/mytestsocket", d.instance.djsonsocket, "djsonsocket")
         assert_equal(1, d.instance.acktimeoutms, "acktimeoutms")
         assert_equal([ "^mdsd.syslog" ], d.instance.mdsd_tag_regex_patterns, "mdsd_tag_regex_patterns")
```

Comment:
this could be the fluentd unit test framework bug in the config string parsing. when [ "^mdsd\\.syslog" ] is given, the returned value is [ "^mdsd.syslog" ] from the unit test. we may need to follow up with fluentd on this. (or maybe latest version has this fix already.)

---
In reply to: [114448281](https://github.com/Azure/fluentd-plugin-mdsd/pull/28#discussion_r114448281) [](ancestors = 114448281)